setup.py

```diff
@@ -30,13 +30,20 @@
 
 long_description = open("README.rst").read()
 
-install_requires = ['pymongo>=3.6,<4']
+install_requires = ['pymongo>=3.7']
 
 tests_require = ['mockupdb>=1.4.0']
 
 if sys.version_info[0] < 3:
     install_requires.append('futures')
```

Comment:
This kind of version checking does not work in setup.py does not actually work because motor uploads a single py2/py3 universal wheel to pypi: https://pypi.org/project/motor/#files.

If the *universal wheel is built* with python 2 then `install_requires` will include `futures`, but if it's built with python 3 then it will not include `futures`. As proof that this doesn't work let's install motor on python 2.7:
```
(venv) ➜  tmp-motor pip install motor
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Collecting motor
  Using cached https://files.pythonhosted.org/packages/28/ca/a2e740ad2822cbf0d2ac4a3b2b48e043142014ab5c8c2129fa5c71c8cdb5/motor-2.0.0-py2.py3-none-any.whl
Collecting pymongo<4,>=3.6 (from motor)
  Using cached https://files.pythonhosted.org/packages/81/ed/ab60269689de46be0964fd93494f9f86a5196802a9156eddc5e5411750c9/pymongo-3.7.2-cp27-cp27m-macosx_10_13_intel.whl
Installing collected packages: pymongo, motor
Successfully installed motor-2.0.0 pymongo-3.7.2
(venv) ➜  tmp-motor pip freeze
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
motor==2.0.0
pymongo==3.7.2
```

There are alternative ways to specify python version specific dependencies but they may or may not work depending on the user's version setuptools or pip: https://hynek.me/articles/conditional-python-dependencies/

I suggest we stop publishing universal wheels and upload one for each supported python version. This is outside the scope of this PR so I'll file a new issue for it.